lib/order.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'menu'
+
+class Order
+
+attr_reader :basket, :menu, :summary
```

Comment:
Don't expose the basket publicly.  You've defined a `summary` method further down, so the `summary` attribute is obsolete